front/src/App.js

```diff
@@ -1,13 +1,40 @@
-import React from "react";
-import "./App.css";
-
-function App ()
-{
-  return (
-    <div className="App">
-      <p>Hello World!</p>
-    </div>
-  );
+import React from 'react';
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import './App.css';
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+
+import Login from "./components/login";
+import SignUp from "./components/signup";
+
+function App() {
+    return (<Router>
+            <div className="App">
+                <nav className="navbar navbar-expand-lg navbar-light fixed-top">
+                    <div className="container">
```

Comment:
Maybe add here this line with applicatio name?
`<label className="nav">LZ - Model Testing</label>`